cluster/core/src/main/resources/reference.conf

```diff
@@ -2,21 +2,25 @@
 lagom.cluster {
 
   # The cluster node will join itself if akka.cluster.seed-nodes is not configured.
-  # In dev-mode this setting will be on, otherwise the default is off. It's possible 
+  # In dev-mode this setting will be on, otherwise the default is off. It's possible
   # to override that by defining akka.cluster.seed-nodes or set this property to off in
   # the application.conf
   join-self = ${lagom.defaults.cluster.join-self}
-  
+
   # The ActorSystem is terminated when the cluster member
   # is removed. The delay is to give ClusterSingleton actors
   # some time to stop gracefully.
   terminate-system-after-member-removed = 10s
-  
+
   # Exit the JVM forcefully when the ActorSystem has been terminated.
```

Comment:
Maybe this should be reworded, because it no longer exits any time the system has been terminated.